docs/aerodromes/classd/Moorabbin.md

```diff
@@ -22,71 +23,184 @@ MB ADC is responsible for the Class D airspace in the MB CTR `SFC` to `A025`.
+Aircraft should report at the standard VFR entry points:
+- **BTO** (to the north)
+- **BAW** (to the west)
+- **CARR** (to the southwest)
+- **GMH** (to the southeast)
+- **ACE** (to the northeast)
+- **SHOAL** (to the southwest)
+
+| Entry Point | RWY 13 | RWY 17 | RWY 31 | RWY 35 |
+|-------------|--------|--------|--------|--------|
+| BTO         | Join final 13R | Join base 17R | Join downwind 31L | Join oblique downwind 35L |
+| BAW         | Join base 13R  | Join base 17R | Join oblique downwind 31L | Join base 35L |
+| CARR        | Join downwind 13R | Join downwind 17R | Join oblique base 31L | Join final 35L |
+| GMH         | Join downwind 13L | Join base 17L | Join oblique base 31R | Join base 35R |
+| ACE         | Join base 13L | Join oblique base 17L | Join base 31R | Join oblique downwind 35R |
+| SHOAL       | Join base 13L | Join base 17L | Join base 31R | Join base 35R |
+
+CTR Entry Altitude: `A010`.
 
 ## Circuits
-Circuits are to be flown at `A010`.
-
-| Runway | Day | Night |
-| ------ | ------ | ----|
-| 13L     | Left  | Left |
-| 13R     | Right | - |
-| 31L     | Left  | - |
-| 31R     | Right  | Right |
-| 17L     | Left  | Left |
-| 17R     | Right | - |
-| 35L     | Left  | - |
-| 35R     | Right  | Right |
 
-## Coordination
+### Fixed wing
+
+Circuit altitude: `A010`.
+
+| Runway  | Day Circuit Direction | Night Circuit Direction |
+|---------|------------------------|--------------------------|
+| 13L     | Left                  | Left                    |
+| 13R     | Right                 | N/A                     |
+| 31L     | Left                  | N/A                     |
+| 31R     | Right                 | Right                   |
+| 17L     | Left                  | Left                    |
+| 17R     | Right                 | N/A                     |
+| 35L     | Left                  | N/A                     |
+| 35R     | Right                 | Right                   |
+
+## Helicopters
+
+### Circuits
+
+#### Day VFR Circuits
+Helicopter circuits are conducted from designated areas based on the active runway:
+
+- **RWY 17/35**: From the **Eastern Grass**, which encompasses all areas east of runway 35R.
+- **RWY 13/31**: From the **Western Triangle**, which extends:
+    - From TWY A to the boundary fence,
+    - Up to the centerline of TWY A4,
+    - Then across to the edge of TWY B and runs parallel to TWY B to the boundary fence.
+
+<figure markdown>
+![Eastern Grass and Western Triangle](img/mbhelicircuitareas.png){ width="700" }
+<figcaption>Eastern Grass and Western Triangle</figcaption>
+</figure>
+
+Circuit operations are conducted parallel to the duty runway and inside the fixed-wing circuit.
+
+**Radio Procedure**:
+
+- Pilots established on the eastern grass or western triangle may operate up to `A001` for low-level airwork.
```

Comment:
To clarify, no clearance is required to conduct these low level ops? I believe at YSBK a clearance must be sought prior.

Reply:
Not when established on the eastern grass or western triangle no. Eg it's not uncommon at mb to do some hovering practice and training in between circuits etc. Essendon does require explicit clearance and often they will just say cleared not above 100 feet.